phovea.js

```diff
@@ -149,7 +149,6 @@ module.exports = function(registry) {
   }, {
     name: 'BoxPlot',
     icon: function() { return System.import('./src/assets/box_icon.png'); },
-    scaling: 'aspect',
```

Comment:
The boxplot previously filled the whole space it was given - the scaling aspect ratio is concerning that space. The aspect ratio was therefore forcing the multiform to change proportions whenever switched to boxplot. There probably would be way to avoid that, but now the boxplot is "smarter". Regardless of the space you give it, it is drawn proportionally and this constraint is not needed anymore. The only exception to proportional scaling is, when the proportional hight falls below certain limit... then it is drawn as high as possible.